convert2rhel/checks.py

```diff
@@ -475,6 +484,7 @@ def check_package_updates():
             "If you wish to update then before continuing with the conversion, execute the following step:\n"
             "Run: yum update -y\n" % (len(packages_to_update), ", ".join(packages_to_update))
```

Comment:
```suggestion
            "Not updating the packages may cause the conversion to fail."
            % (len(packages_to_update), " ".join(packages_to_update))
```
I'd expect linux admins to know ho to update packages.
Also I removed the "," between the listed packages so they can copy and paste it to the yum update command.

Reply:
done.